ftw/subsite/browser/configure.zcml

```diff
@@ -50,7 +50,6 @@
       />
 
   <browser:viewlet
-      layer="ftw.subsite.interfaces.IFtwSubsiteLayer"
```

Comment:
Why do you delete that one? I should be replaces by another layer, otherwise it will be active even thought the ftw.subsite profile is not installed.

Reply:
I can't remember, I thinks we were discussing this already. I added it again due tu the fact that I can't think of a reason it shouldn't be there.